<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">;
border-image: url(:/new/prefix1/ui/push2.png);</string>
   </property>
   <property name="text">
    <string>login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accountEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/ui/frame3.png);</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>记住用户名和密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/ui/frame3.png);</string>
   </property>
  </widget>
  <widget class="QToolButton" name="closetoolButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/ui/exit.png);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="mintoolButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>247</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/ui/push2.png);</string>
   </property>
   <property name="text">
    <string>register</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-290</x>
     <y>-840</y>
     <width>2551</width>
     <height>1691</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/ui/201909271220430vlYm.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>OurDBMS</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>loginButton</zorder>
  <zorder>accountEdit</zorder>
  <zorder>checkBox</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>passwordEdit</zorder>
  <zorder>closetoolButton</zorder>
  <zorder>mintoolButton</zorder>
  <zorder>rButton</zorder>
  <zorder>label_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
